### PR TITLE
remove more BBB code

### PR DESCRIPTION
**Pull request by thet**
This ones removes the BBB import `from plone.testing._z2_testbrowser import Browser` from `z2.py`. It's "BBB" comment makes me think, this should not be used anyways. Also, _z2_testbrowser imports some internals from zope.testbrowser, which are removed in zope.testbrowser 5.0 (e.g. `zope.testbrowser.testing.PublisherConnection`)

I'll again run the testsuite, which I expect to fail because of plone.testing 5.0 changes to fail with incorrect use of testing layers.

Should we rather deprecate before removing anything?

/cc @do3cc

### CODE REVIEW

**Comment by thet** [comment]
Regardin the Changelog: There is already a "remove BBB code" changelog entry - no need for another one.

**Comment by thet** [comment]
ok, we need to provide an alternative import for ``z2.Browser``.

**Comment by jensens** [comment]
+1 for deprecation first

**Comment by thet** [comment]
@jensens for some reasons the following doesn't work when running tests. Or did I use zope.deferredimport wrong?
```
import zope.deferredimport
zope.deferredimport.initialize()

zope.deferredimport.deprecated(
    "Import from zope.testbrowser.browser instead",
    Browser='zope.testbrowser:browser.Browser',
)
```

**Comment by thet** [comment]
... but zope.testbrowser.browser.Browser isn't a alternative for the old browser from here. have to find something else...

**Comment by jensens** [comment]
anyway, iirc after ``:`` the class and not a dotted-path is expected.

File: src/plone/testing/_z2_testbrowser.py [hunk 1]
```diff
@@ -129,8 +27,8 @@ def saveState(func):
     """
     from AccessControl.SecurityManagement import getSecurityManager
     from AccessControl.SecurityManagement import setSecurityManager
-    from zope.component.hooks import getSite
-    from zope.component.hooks import setSite
+    from zope.site.hooks import getSite
+    from zope.site.hooks import setSite
```
Comment (thet) [inline src/plone/testing/_z2_testbrowser.py]: the imports from zope.site are deprecated: https://github.com/zopefoundation/zope.site/blob/master/src/zope/site/hooks.py

**Pull request merged**